packages/client/utils/constants.ts

```diff
@@ -160,10 +161,16 @@ export const SPOTLIGHT_TOP_SECTION_HEIGHT = 236
 
 export const PARABOL_AI_USER_ID = 'parabolAIUser'
 
+export const StarterBenefits = [
+  `${Threshold.MAX_STARTER_TIER_TEAMS} teams`,
+  'Retrospectives, Sprint Poker, Standups, Check-Ins',
+  'Unlimited templates',
```

Comment:
+1 I'm not sure if this is self explanatory without knowing the history of this limit. Maybe something like "All predefined templates", "Parabol's premium templates" or just "Meeting templates"?

Reply:
I think it's similar to the "Unlimited team member" benefit that we also list in the Starter tier. I've updated it to "Unlimited meeting templates" to clarify what type of template it is